sklearn/manifold/_mds.py

```diff
@@ -283,24 +345,29 @@ class MDS(BaseEstimator):
         Number of dimensions in which to immerse the dissimilarities.
 
     metric : boolean, optional, default: True
+        If  ``method=='svd'``, metric must be set to True.
```

Comment:
We usually describe the parameter first and only then explain special cases like this. I.e. this should be the last sentence.

Same for the other arguments please

Reply:
Done